views/find_missing.py

```diff
@@ -56,6 +75,7 @@ def render_model_field_tree(self):
 
         self.model_field_chooser = model_field_tree.ModelFieldTree(mw, self.model_field_items)
         self.model_field_items = self.model_field_chooser.all_items
+        self.model_field_selected_items = self.model_field_chooser.get_all_items(True)
```

Comment:
We should call `self.update_init_search()` here too, if the search preview is to stay visible. I have to toggle on and off the model for the search query to visibly update. Deck chooser is a little out of our control though, not sure if we can listen to one of its events.

Reply:
Yea good call. Done.